### PR TITLE
docs: update English and Chinese versions of readme

### PR DESCRIPTION
**Pull request by lumianph**
it still depends on two todo jobs, marked as TBD in the PR

### CODE REVIEW

**Comment by lumianph** [comment]
[docs reorganize, review and udpate](https://trello.com/c/EW0F7yOb/1-docs-reorganize-review-and-udpate)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,69 @@
...
+* Under specific circumstances, unexpected outcomes produced by SQL queries with the WHERE when certain WHERE conditions do not fit into indexes https://github.com/4paradigm/OpenMLDB/issues/599
+* The bug when enabling WindowParallelOpt and WindowSkewOptimization at the same times https://github.com/4paradigm/OpenMLDB/issues/444
+* The bug of LCA (Lowest Common Ancestor) algorithm to support WindowParallelOpt for particular SQLs https://github.com/4paradigm/OpenMLDB/issues/485
+* Workaround for the Spark bug (SPARK-36932) when the columns with the same names in LastJoin https://github.com/4paradigm/OpenMLDB/issues/484
+### Acknowledgement
+We appreciate the contribution to this release from external contributors who are not from 4Paradigm's core OpenMLDB team, including [Kanekanekane](https://github.com/Kanekanekane), [shawn-happy](https://github.com/shawn-happy), [lotabout](https://github.com/lotabout), [Shouren](https://github.com/Shouren), [zoyopei](https://github.com/zoyopei), [huqianshan](https://github.com/huqianshan)
+
 ## [Unreleased]
```
Comment (aceforeverd) [inline CHANGELOG.md]: Unreleased tag should always in top

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,69 @@
 # Changelog
 
+## [0.4.0] - 2022-01-14
```
Comment (aceforeverd) [inline CHANGELOG.md:3]: need update links in line 126 to make those links work

checkout https://keepachangelog.com/en/1.0.0/

Comment (aceforeverd) [inline CHANGELOG.md:3]: need update links in line 126 to make those links work

checkout https://keepachangelog.com/en/1.0.0/

File: CHANGELOG.md (hunk 1 above)
Comment (lumianph) [reply CHANGELOG.md]: does this unreleased make sense to us? are those functions merged into our new version?

Comment (lumianph) [reply CHANGELOG.md]: i have removed the unreleased tag for now

Comment (aceforeverd) [reply CHANGELOG.md]: I think in practice the content of `[unreleased]` should updated during development phase if there is something important to notice e.g new SQL clause HAVING.  And revised before final release, rename release to a version number and create a new `[unreleased]` tag. Just as keepachangelog.com says. 

Generally we forgot to update CHANGELOG during 0.4.0 development phase.

Comment (aceforeverd) [reply CHANGELOG.md]: Those unreleased content was in 0.3.0, it is not updated for a while

Comment (lumianph) [reply CHANGELOG.md]: yes, agree. so i remove the released tag for now. i checked some projects, some of them also don't have this section. so i consider it insignificant

**Review by aceforeverd — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**